### PR TITLE
bugfix in setDefaultFilter - revert from 5.x

### PR DESCRIPTION
**Pull request by petrparolek**

### CODE REVIEW

File: src/DataGrid.php [hunk 1]
```diff
@@ -1360,22 +1360,32 @@ public function setFilter(array $filter): self
...
 			$filter = $this->getFilter($key);
 
+			if ($filter === null) {
+				throw new DataGridException("Can not set default value to nonexisting filter [$key]");
+			}
+
 			if ($filter instanceof FilterMultiSelect && !is_array($value)) {
 				throw new DataGridException(
-					sprintf('Default value of filter [%s] - MultiSelect has to be an array', $key)
+					"Default value of filter [$key] - MultiSelect has to be an array"
```
Comment (mabar) [inline src/DataGrid.php]: Why you feel like coding style changes are needed?

Comment (petrparolek) [reply src/DataGrid.php]: fixed

**Comment by petrparolek** [comment]
PHPCs doesn't like empty(), I don't know why

**Comment by mabar** [comment]
empty() may lead to unexpected results, that's why we have it disabled. It's better to be explicit and check empty array `$var === []`

**Comment by petrparolek** [comment]
@paveljanda  @f3l1x , ould you please review PR and merge it?

**Comment by petrparolek** [comment]
@paveljanda  is bug persist now?

**Comment by paveljanda** [comment]
Thanks! :)

**Pull request merged**